src/components/Sections/PartnersSection/PartnersSection.tsx

```diff
@@ -0,0 +1,17 @@
+import { useContext } from 'react';
+import { LanguageModeContext } from '../../../contexts/LanguageContext';
+import './PartnersSection.scss';
+
+const PartnersSection = () => {
+  const languageMode = useContext(LanguageModeContext);
+
+  return (
+    <>
+      <div id="partners-header-container">
```

Comment:
skoro tu jest div który jest kontenerem to po co używać <> w 9 linijce jako wraper?

Reply:
Racja, zapomimało mi się